portfolio/src/main/webapp/style.css

```diff
@@ -186,6 +184,92 @@ p {
   text-align: left;
 }
 
+.toggle-comment-section {
+  border: none;
+  width: 10%;
```

Comment:
Usually we do this with a fixed size the rest stretchable by width: calc(100% - fixedpx)